prov/efa/src/efa_domain.c

```diff
@@ -496,7 +496,10 @@ int efa_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 err_free_info:
 	fi_freeinfo(domain->info);
 err_close_domain:
-	ofi_domain_close(&domain->util_domain);
+	if (ofi_domain_close(&domain->util_domain)) {
```

Comment:
Can you save the err code, and print it?

Reply:
Sure